workbooks/views.py

```diff
@@ -672,15 +672,13 @@ def workbook_create_with_plot(request):
 def worksheet_plots(request, workbook_id=0, worksheet_id=0, plot_id=0):
     command = request.path.rsplit('/', 1)[1];
     json_response = False
-    default_name = "Untitled Workbook"
     result = {}
     try:
         workbook_model = Workbook.objects.get(id=workbook_id) if workbook_id else None
 
         if request.method == "POST":
             workbook_model.save()
             if command == "delete":
-                var = Worksheet_plot.objects.get(id=plot_id).delete()
                 result['message'] = "This plot has been deleted from workbook."
```

Comment:
This call needs to be reinstated, though the variable assignment can be removed.